src/test/scala/org/ergoplatform/nodeView/history/BlockSectionValidationSpecification.scala

```diff
@@ -24,9 +56,9 @@ class BlockSectionValidationSpecification extends HistoryTestHelpers {
     commonChecks(history, block.extension, block.header)
   }
 
-  private def init() = {
+  private def init(version: Version = Header.InitialVersion) = {
```

Comment:
I suggest to always pass version explicitly.

Reply:
for testing code this is okay I guess